### PR TITLE
#3407 optimize UI for showing join/decline button

### PR DESCRIPTION
**Pull request by yoution**

### CODE REVIEW

File: src/components/Invitation/Invitation.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+ */
+import React from 'react'
+import PT from 'prop-types'
+
+import LoadingIndicator from '../LoadingIndicator/LoadingIndicator'
+import cn from 'classnames'
+import styles from './Invitation.scss'
+import ArrowIcon from '../../assets/icons/arrow-9px-carret-down-normal.svg'
+import CheckIcon from '../../assets/icons/ui-check.svg'
+console.log(styles)
```
Comment (maxceem) [inline src/components/Invitation/Invitation.jsx]: Could you please remove `console.log`.

**Comment by yoution** [comment]
@maxceem  
image

I don't know why the tick  not show correctly? maybe some the id conflict in the svg file, I don't know how to solve it

**Comment by maxceem** [comment]
> I don't know why the tick not show correctly? maybe some the id conflict in the svg file, I don't know how to solve it

@yoution I don't remember in which can did I see such a bug. If this is the only one issue left, please, submit and will check it out.

**Comment by yoution** [comment]
@maxceem please review. https://github.com/appirio-tech/connect-app/pull/3518

**Comment by yoution** [comment]
I donot know why，in the previous pushed，the tick shows correctly in list page，but shows incorrectly in grid page, but they use the same conponent and same svg file

**Comment by maxceem** [comment]
> I donot know why，in the previous pushed，the tick shows correctly in list page，but shows incorrectly in grid page, but they use the same conponent and same svg file

It looks good for me on both pages grid and card. Maybe you try to stop `npm start` and run it again?

**Pull request merged**